requirements-dev.txt

```diff
@@ -9,7 +9,7 @@ pytest-forked>=1.4.0
 pytest-benchmark
 pytest-split
 black>=21
-tomli~=2.0.0 # Drop once minimum Python version is 3.11
+tomlkit~=0.13
```

Comment:
We actually can't, because the Python native `tomllib` is also read only, and we need the write ability of `tomlkit` to save a configuration file. See `configuration.py`.